dnd5esheets/repositories/base.py

```diff
@@ -46,3 +55,19 @@ async def delete_by_id(cls, session: AsyncSession, id: int):
         model = await cls.get_by_id(session, id=id)
         await session.delete(model)
         await session.commit()
+
+    @classmethod
+    async def _search(
+        cls, session: AsyncSession, search_term: str, sort_by_rank: bool = True
+    ) -> Sequence[Row[Any]]:
+        fields = ["rank"] + cls.search_fields
+        query = (
+            f"SELECT {', '.join(fields)} FROM {cls.search_index_table_name()} "
+            f"WHERE {cls.search_index_table_name()} MATCH :search_term "
+            f"{'ORDER BY rank' if sort_by_rank else ''};"
```

Comment:
I don't see `rank` in the `item_search_index`. Is this something populated by fts5?